mu4/domain/importexport/internal/musicxml/exportxml.cpp

```diff
@@ -5053,18 +5054,43 @@ static bool elementRighter(const Element* e1, const Element* e2)
 //  measureStyle -- write measure-style
 //---------------------------------------------------------
 
-// this is done at the first measure of a multi-meaure rest
+// this is done at the first measure of a multimeasure rest
 // note: for a normal measure, mmRest1 is the measure itself,
 // for a multi-meaure rest, it is the replacing measure
 
-static void measureStyle(XmlWriter& xml, Attributes& attr, const Measure* const m)
+static void measureStyle(XmlWriter& xml, Attributes& attr, const Measure* const m, const int partIndex)
 {
     const Measure* mmR1 = m->mmRest1();
     if (m != mmR1 && m == mmR1->mmRestFirst()) {
         attr.doAttr(xml, true);
         xml.stag("measure-style");
         xml.tag("multiple-rest", mmR1->mmRestCount());
         xml.etag();
+    } else {
+        Part* part = m->score()->parts().at(partIndex);
+        for (int i = 0; i < part->nstaves(); ++i) {
+            int staffIdx = part->staff(i)->idx();
```

Comment:
Probably there is some slightly more clever way of dealing with this - if you look at the implementation of idx(), you'll see it's relying on indexOf, which is probably slow operation.  Not that it really matters, and I know I told you this was the way to do it.  But still, you asked for feedback, that's what I've got... :-)

Reply:
Improved efficiency by only calling it once, before the loop commences.